scripts/import_base.py

```diff
@@ -5,21 +5,21 @@
 from github import Github
 import os
 
+
 class RVDImport:
 
     def __init__(self):
         # Authentication for user filing issue (must have read/write access to
         # repository to add issue to)
         try:
-            self.token = os.environ['GITHUB_TOKEN'] #ffb0d091869e647e3db4baa4d71dcb5c3c6a17d7
+            self.token = os.environ['GITHUB_TOKEN']  # ffb0d091869e647e3db4baa4d71dcb5c3c6a17d7
```

Comment:
This token is invalid and was placed here as a self-reference, suggesting to remove it
```suggestion
            self.token = os.environ['GITHUB_TOKEN'] 
```